client.cpp

```diff
@@ -985,7 +985,8 @@ void crc_verify_client::handle_response(struct timeval timestamp, request *reque
         m_stats.update_get_latency_map(response->get_latency());
     }
 
-    if (response->is_error() || !values_count) {
+    if (strcmp(response->get_status(), "PROTOCOL_BINARY_RESPONSE_KEY_ENOENT") == 0 ||
+                                                response->is_error() || !values_count) {
```

Comment:
should not we also raise "failed to verify" counter?

Reply:
@sashas-lb the "failed to verified" stats is related to the errors count and this will increment the error counter